data/store/mongo/suite_store_test.go

```diff
@@ -0,0 +1,24 @@
+package mongo_test
```

Comment:
For consistency sake, please rename to `mongo_suite_test.go`. It is `<package>_suite_test.go`.